### ISSUE TITLE
Retain Introns, start codons, and stop codons in gff3 file?

### ISSUE THREAD

**Issue by christopher-holt** [jorvis/Attributor#5]
Would it be possible to add an option that would allow the gff3 file from attributor to retain the introns, start codons, and stop codons? Thanks!

**Comment by jorvis** [comment]
When you say to retain them, do you mean to have specific feature entries in the GFF file for each?  Features like introns, of course, can usually be derived from existing features, but here are you asking for them to be explicit?

**Comment by christopher-holt** [comment]
Yes, I was thinking explicit feature entires if that would be possible?

**Comment by jorvis** [comment]
Certainly doable, I just am not sure how soon I'll be able to work on it. (We are in major conference/workshop prep mode for a few weeks.)  I'll keep this open, and can certainly review any pull requests.
